Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml

```diff
@@ -88,6 +88,115 @@
       countdown_sound:
         path: /Audio/Effects/minibombcountdown.ogg
 
+- type: entity
+  name: supermatter grenade
+  description: Grenade that simulates delamination of supermatter engine, pulling things in heap and explode after some time.
+  parent: GrenadeBase
+  id: SupermatterGrenade
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Grenades/supermattergrenade.rsi
+  - type: OnUseTimerTrigger
+    delay: 3
```

Comment:
should probably add multiple settings, only having 3 seconds is awful
also the uplink description is completely off so fix it or remove it if adding multiple settings

Reply:
What?
What settings?

Reply:
you can have multiple timer options, see c4

Reply:
SO THIS IS "delayOptions" FOR.
OMG, fucking undocumented shit

Reply:
Anyway, this timer is before gravity well starts, not before explosion

Reply:
oh then its fine to be short